1-js/01-getting-started/4-devtools/article.md

```diff
@@ -1,63 +1,63 @@
-# Developer console
+# Entwickler-Konsole
 
-Code is prone to errors. You will quite likely make errors... Oh, what am I talking about? You are *absolutely* going to make errors, at least if you're a human, not a [robot](https://en.wikipedia.org/wiki/Bender_(Futurama)).
+Code ist anfällig für Fehler. Du wirst sehr wahrscheinlich Fehler machen... Oh, was rede ich da? Du wirst *auf jeden Fall* Fehler machen, zumindest wenn du ein Mensch bist und kein [Roboter](https://en.wikipedia.org/wiki/Bender_(Futurama)).
```

Comment:
Maybe change the wiki page to the german page

Reply:
There is currently no corresponding German wiki page for it.